lib/rucio/daemons/transmogrifier/transmogrifier.py

```diff
@@ -146,6 +147,41 @@ def is_matching_subscription(subscription, did, metadata):
     return True
 
 
+def select_algorithm(algorithm, rule_ids, params):
+    """
+    Method used in case of chained subscriptions
+
+    :param algorithm: Algorithm used for the chained rule. Now only associated_site
+                      associated_site : Choose an associated endpoint according to the RSE attribute assoiciated_site
+    :param rule_ids: List of parent rules
+    :param params: List of rules parameters to be used by the algorithm
+    """
+    selected_rses = {}
+    if algorithm == 'associated_site':
```

Comment:
We should right away go for a model similar to the lfn2pfn or surl algorithms, so we can push this stuff of to the community modules.